### PR TITLE
Tweak font size/tweak font weight shortcut in text edit mode

### PR DESCRIPTION
**Pull request by bkrmendy**
Make the set font size/set font weight strategies work in text edit mode as well

### CODE REVIEW

File: editor/src/components/text-editor/text-editor.spec.browser2.tsx [hunk 1]
```diff
@@ -209,6 +209,41 @@ describe('Use the text editor', () => {
       expect(after).toEqual(projectWithStyle('textDecoration', 'none'))
     })
 
+    it('supports increasing font size', async () => {
+      const { before, after } = await testModifier(
+        { shift: true, cmd: true, alt: false, ctrl: false },
```
Comment (ruggi) [inline editor/src/components/text-editor/text-editor.spec.browser2.tsx:214]: you can use `shitCmdModifier` here

File: editor/src/components/text-editor/text-editor.spec.browser2.tsx [hunk 2]
```diff
@@ -209,6 +209,41 @@ describe('Use the text editor', () => {
...
+      const { before, after } = await testModifier(
+        { shift: false, cmd: true, alt: true, ctrl: false },
+        '.',
+      )
+      expect(before).toEqual(projectWithStyleNoQuotes('fontWeight', '500'))
+      expect(after).toEqual(projectWithStyleNoQuotes('fontWeight', '600'))
+    })
+    it('supports decreasing font size', async () => {
+      const { before, after } = await testModifier(
+        { shift: true, cmd: true, alt: false, ctrl: false },
```
Comment (ruggi) [inline editor/src/components/text-editor/text-editor.spec.browser2.tsx:231]: `shiftCmdModifier` as well

File: editor/src/components/text-editor/text-editor.spec.browser2.tsx [hunk 3]
```diff
@@ -209,6 +209,41 @@ describe('Use the text editor', () => {
...
+        '.',
+      )
+      expect(before).toEqual(projectWithStyle('fontSize', '17px'))
+      expect(after).toEqual(projectWithStyle('fontSize', '18px'))
+    })
+
+    it('supports increasing font weight', async () => {
+      const { before, after } = await testModifier(
+        { shift: false, cmd: true, alt: true, ctrl: false },
```
Comment (ruggi) [inline editor/src/components/text-editor/text-editor.spec.browser2.tsx:223]: worth adding an exported `altCmdModifier` or `optCmdModifier` and use it here

File: editor/src/components/text-editor/text-editor.spec.browser2.tsx [hunk 4]
```diff
@@ -209,6 +209,41 @@ describe('Use the text editor', () => {
...
+        { shift: true, cmd: true, alt: false, ctrl: false },
+        ',',
+      )
+      expect(before).toEqual(projectWithStyle('fontSize', '15px'))
+      expect(after).toEqual(projectWithStyle('fontSize', '14px'))
+    })
+
+    it('supports decreasing font weight', async () => {
+      const { before, after } = await testModifier(
+        { shift: false, cmd: true, alt: true, ctrl: false },
```
Comment (ruggi) [inline editor/src/components/text-editor/text-editor.spec.browser2.tsx:240]: same here

**Review by ruggi — APPROVED** [review]

File: editor/src/components/text-editor/text-editor.spec.browser2.tsx (hunk 3 above)
Comment (bkrmendy) [reply editor/src/components/text-editor/text-editor.spec.browser2.tsx:223]: done!

File: editor/src/components/text-editor/text-editor.spec.browser2.tsx (hunk 2 above)
Comment (bkrmendy) [reply editor/src/components/text-editor/text-editor.spec.browser2.tsx:231]: done!

File: editor/src/components/text-editor/text-editor.spec.browser2.tsx (hunk 1 above)
Comment (bkrmendy) [reply editor/src/components/text-editor/text-editor.spec.browser2.tsx:214]: done!

File: editor/src/components/text-editor/text-editor.tsx [hunk 5]
```diff
@@ -49,6 +62,68 @@ const handleShortcut = (
...
+): Array<ApplyCommandsAction> => {
+  const modifiers = Modifier.modifiersForEvent(event)
+  const character = keyCharacterFromCode(event.keyCode)
+  const matches = isAdjustFontSizeShortcut(modifiers, character)
+
+  if (!matches) {
+    return []
+  }
+
+  const direction: { [key: string]: number } = { comma: -1, period: 1 }
```
Comment (Rheeseyb) [inline editor/src/components/text-editor/text-editor.tsx:78]: This use of an object is adding a potentially unnecessary `undefined` into the mix later on. Please can you replace this with explicit checks for either of those characters, and return `[]` if we have somehow entered this code for any other keyboard character

**Review by Rheeseyb — APPROVED** [review]

File: editor/src/components/text-editor/text-editor.tsx (hunk 5 above)
Comment (bkrmendy) [reply editor/src/components/text-editor/text-editor.tsx:78]: fixed in https://github.com/concrete-utopia/utopia/pull/3077/commits/ec9f379a406f74b0812f5a94524778619f2a035e

**Pull request merged**